library/src/compute/foundations.rs

```diff
@@ -115,6 +123,90 @@ pub fn assert(
     Value::None
 }
 
+/// Ensure that two values are equal.
+///
+/// Fails with an error if the first value (expected) is not equal to the
+/// second (actual). Does not produce any output in the document.
+///
+/// ## Example
+/// ```example
+/// #assert.eq(10, 10)
+/// ```
+///
+/// Display: Assert Equals
+/// Category: foundations
+/// Returns:
+#[func]
+pub fn assert_eq(
+    /// The expected value.
```

Comment:
I wouldn't call one "expected" and one "actual". I, for one, more frequently write them in the other order in Rust. Rust just calls them left and right. This of course also affects the error message.

Reply:
Alright! Is the new message fine?